unstanle test TestGetSessionWithFault

```
=== NAME  TestGetSessionWithFault
    testing.go:1399: race detected during execution of test
    session_integration_test.go:199: 
        	Error Trace:	pkg/ttl/ttlworker/session_integration_test.go:199
        	            				pkg/ttl/ttlworker/session.go:76
        	            				pkg/session/syssession/session.go:519
        	            				pkg/session/syssession/session.go:278
        	            				pkg/session/syssession/session.go:518
        	            				pkg/ttl/ttlworker/session.go:68
        	            				pkg/ttl/ttlworker/session_integration_test.go:162
        	            				pkg/session/syssession/pool.go:268
        	            				pkg/ttl/ttlworker/session_integration_test.go:147
        	            				pkg/ttl/ttlworker/session.go:67
        	            				pkg/ttl/ttlworker/job_manager_test.go:151
        	            				pkg/ttl/ttlworker/session_integration_test.go:243
        	Error:      	Expected nil, but got: &ttlworker_test.mockAttached{SQLExecutor:(*session.session)(0xc007680008)}
        	Test:       	TestGetSessionWithFault
=== NAME  TestGetSessionWithFault/set_tidb_retry_limit=0
    testing.go:1577: test executed panic(nil) or runtime.Goexit: subtest may have called FailNow on a parent test
--- FAIL: TestGetSessionWithFault/set_tidb_retry_limit=0 (0.00s)
```

--- a/pkg/ttl/ttlworker/job_manager_integration_test.go
+++ b/pkg/ttl/ttlworker/job_manager_integration_test.go
@@ -108,7 +108,11 @@ func TestWithSession(t *testing.T) {
 		ttlworker.DetachStatsCollector = origDetachStats
 	}()
 
-	store := testkit.CreateMockStore(t)
+	store, dom := testkit.CreateMockStoreAndDomain(t)
+	// stop TTLJobManager to avoid unnecessary job schedule and make test stable
+	dom.TTLJobManager().Stop()
+	require.NoError(t, dom.TTLJobManager().WaitStopped(context.Background(), time.Minute))
+
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("set @@time_zone = 'Asia/Shanghai'")
 	tk.MustExec("set @@global.time_zone= 'Europe/Berlin'")

--- a/pkg/ttl/ttlworker/session_integration_test.go
+++ b/pkg/ttl/ttlworker/session_integration_test.go
@@ -180,6 +180,10 @@ func TestGetSessionWithFault(t *testing.T) {
 	}()
 
 	_, dom := testkit.CreateMockStoreAndDomain(t)
+	// stop TTLJobManager to avoid unnecessary job schedule and make test stable
+	dom.TTLJobManager().Stop()
+	require.NoError(t, dom.TTLJobManager().WaitStopped(context.Background(), time.Minute))
+
 	pool := newFaultSessionPool(t, dom.AdvancedSysSessionPool())
 
 	var sysSe *syssession.Session

